ansible/roles_ocp_workloads/ocp4_workload_sealed_secrets/tasks/workload.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Implement your Workload deployment tasks here
+
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+- name: Install Operator
+  include_role:
+    name: install_operator
+  vars:
+    install_operator_action: install
+    install_operator_name: sealed-secrets-operator-helm
+    install_operator_namespace: sealed-secrets
+    install_operator_channel: "{{ ocp4_workload_sealed_secrets_channel }}"
+    install_operator_catalog: community-operators
+    install_operator_automatic_install_plan_approval: "{{ ocp4_workload_sealed_secrets_automatic_install_plan_approval | default(true) }}"
+    install_operator_starting_csv: "{{ ocp4_workload_sealed_secrets_starting_csv }}"
+    install_operator_use_catalog_snapshot: "{{ ocp4_workload_sealed_secrets_use_catalog_snapshot | default(false)}}"
+    install_operator_catalogsource_name: "{{ ocp4_workload_sealed_secrets_catalogsource_name | default('') }}"
+    install_operator_catalog_snapshot_image: "{{ ocp4_workload_sealed_secrets_catalog_snapshot_image | default('') }}"
+    install_operator_catalog_snapshot_image_tag: "{{ ocp4_workload_sealed_secrets_catalog_snapshot_image_tag | default('') }}"
+
+- name: Deploy Sealed Secrets Controller
+  k8s:
+    state: present
+    definition: "{{ lookup('file', 'sealed-secrets-controller.yaml' ) | from_yaml }}"
```

Comment:
This can be simplified using `src`:

```
- name: Deploy Sealed Secrets Controller
  k8s:
    state: present
    src: sealed-secrets-controller.yaml
```